### PR TITLE
Hobbies editing

### PR DESCRIPTION
**Pull request by alikew24**
Edited the hobbies.html page, adding images and blurbs about each one.

### CODE REVIEW

File: portfolio/src/main/webapp/hobbies.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  <head>
+    <meta charset="UTF-8">
+    <title>Hobbies</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+      <div class="content">
+          <h1> Hobbies </h1>
+          <nav>
```
Comment (estep-google) [inline portfolio/src/main/webapp/hobbies.html:12]: Nice!!

File: portfolio/src/main/webapp/hobbies.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            <ul id="navigation">
+                <li><a href="index.html"> Home </a></li>
+                <li><a href="projects.html"> Projects </a></li>
+                <li><a href="hobbies.html"> Hobbies </a></li>
```
Comment (estep-google) [inline portfolio/src/main/webapp/hobbies.html:16]: You shouldn't have whitespace between the tags and the content.

```
<li><a href="index.html">Home</a></li>
```

File: portfolio/src/main/webapp/hobbies.html [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+            </ul>
+          </nav>
+          <div class="alt1">
+            <h2> Hiking </h2>
+            <img class="project-image" src="images/hike.jpg"/>
+            <p> I love the outdoors and hiking! Last summer, I lived in the Adirondack Mountains in NY, and
```
Comment (estep-google) [inline portfolio/src/main/webapp/hobbies.html:22]: Please reformat paragraph tags like this:

```
<p>
  Text ...
  More text...
</p>
```

File: portfolio/src/main/webapp/hobbies.html [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+            <img class="project-image" src="images/climb.jpg"/>
+            <p> Although I used to be afraid of heights, rock climbing is now one of my favorite hobbies. Cornell
+                has a rock wall on campus that I belong to, and love to top rope and boulder there! </p>
+           </div>
+           <div class="alt1">
+                <h2> Running </h2>
+                <img class="project-image" src="images/run.jpg"/>
+                <p> I have a love-hate relationship with running. Although sometimes my laziness gets the best of me, 
+                    I love to run outdoors, especially when it is raining. This photo was taken at the Central Park Half Marathon 
+                    in New York. I hope to run a full marathon one day! </p>
```
Comment (estep-google) [inline portfolio/src/main/webapp/hobbies.html:37]: Use consistent indenting (2 spaces looks like your rule here (which is great in my opinion)). There are multiple places where the indenting is off.

File: portfolio/src/main/webapp/style.css [hunk 5]
```diff
@@ -10,6 +10,25 @@ nav {
...
+  margin-left: auto;
+  margin-right: auto;
+  width: 650px;
+  text-align: center;
+}
+nav {
+    list-style-type: none;
+    background-color: lightblue;
+    border: solid;
+    margin-bottom: 20px;
```
Comment (estep-google) [inline portfolio/src/main/webapp/style.css:23]: Consistent indenting again (2 spaces remains my recommendation; however, it should match what you use in all of your files, regardless of type).

Also line up your block definitions to be at the start of each line.

File: portfolio/src/main/webapp/style.css [hunk 6]
```diff
@@ -10,6 +10,25 @@ nav {
...
 }
+.content {
+  margin-left: auto;
+  margin-right: auto;
+  width: 650px;
+  text-align: center;
+}
+nav {
+    list-style-type: none;
+    background-color: lightblue;
```
Comment (estep-google) [inline portfolio/src/main/webapp/style.css:21]: Please alphabetize CSS properties in each block.

File: portfolio/src/main/webapp/hobbies.html [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+      <div class="content">
+          <h1> Hobbies </h1>
+          <nav>
+            <ul id="navigation">
+                <li><a href="index.html"> Home </a></li>
+                <li><a href="projects.html"> Projects </a></li>
+                <li><a href="hobbies.html"> Hobbies </a></li>
+            </ul>
+          </nav>
+          <div class="alt1">
```
Comment (estep-google) [inline portfolio/src/main/webapp/hobbies.html:19]: I couldn't (for reasons unknown to me) couldn't comment on the CSS class names in the CSS file.

I don't love `alt1` and `alt2` as class names. They're not very descriptive. You're doing this for alternating background colors for your rows (yay!). I'm not quite sure what the right name is here, but something that references the fact that this is a row background class.

Comment (estep-google) [reply portfolio/src/main/webapp/hobbies.html:19]: Actually, I picked this up from a comment I saw from Daniel elsewhere.

You can use one class on each `<div>` for a hobby (`hobby-item`?). In CSS:

```
.hobby-item:nth-of-type(odd) {
  background-color: #0000ff;
}

.hobby-item:nth-of-type(even) {
  background-color: #00ff00;
}
```

**Comment by alikew24** [comment]
added the edits!

File: portfolio/src/main/webapp/hobbies.html [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+            sea animals, and everything about scuba diving, and I can't wait to travel more and dive in new places.
+          </p>
+        </div>
+        <footer>
+          <ul id="footer">
+            <li><a href="https://linkedin.com/in/ali-kew">LinkedIn</a></li>
+            <li><a href="https://github.com/alikew24">Github</a></li>
+          </ul>
+        </footer>
+      </div>
```
Comment (estep-google) [inline portfolio/src/main/webapp/hobbies.html:67]: I can't remember the exact stlyistic pattern here, but I think a comment on the closing `<div>` is in order here. It reminds readers what large section of code is being closed. Something like `</div>  <!-- #content -->` (notice two spaces between the end of `<div>` and the start of `<!--`).

Basically, if I have to scroll a fair distance to try to guess what tag this one is closing, a comment is likely in order.

**Comment by alikew24** [comment]
Gotcha- just added that.

**Pull request merged**